src/main/resources/joant_jcrOAuthView/html/jcrOAuthView.jsp

```diff
@@ -89,13 +89,20 @@
                 <div flex="45" layout="row">
                     <md-input-container flex>
                         <label message-key="joant_jcrOAuthView.label.fieldFromMapper"></label>
-                        <md-select ng-model="mapped.mapper" ng-model-options="{trackBy: '$value.name'}">
+                        <md-select ng-if="!mapped.editable" ng-model="mapped.mapper" ng-model-options="{trackBy: '$value.name'}">
                             <md-optgroup>
                                 <md-option ng-repeat="mapperProperty in jcrOAuthProvider.mapperProperties | selectable:{mapping:jcrOAuthProvider.mapping,key:'mapper',selected:mapped.mapper} | typeMatch:mapped.connector.valueType | orderBy:jcrOAuthProvider.orderByMapper" ng-value="mapperProperty" val="{{mapperProperty.name}}">
                                     {{ jcrOAuthProvider.getMapperI18n(mapperProperty.name) }} <span ng-if="mapperProperty.mandatory" class="joa-mandatory-property" message-key="joant_jcrOAuthView.label.mandatory"></span>
                                 </md-option>
                             </md-optgroup>
                         </md-select>
+                        <md-select ng-if="mapped.editable" ng-model="mapped.mapper" ng-model-options="{trackBy: '$value.name'}">
+                            <md-optgroup>
+                                <md-option ng-repeat="mapperProperty in jcrOAuthProvider.mapperProperties | selectable:{mapping:jcrOAuthProvider.mapping,key:'mapper',selected:mapped.mapper} | orderBy:jcrOAuthProvider.orderByMapper" ng-value="mapperProperty" val="{{mapperProperty.name}}">
+                                    {{ jcrOAuthProvider.getMapperI18n(mapperProperty.name) }} <span ng-if="mapperProperty.mandatory" class="joa-mandatory-property" message-key="joant_jcrOAuthView.label.mandatory"></span>
```

Comment:
Don't we have some property where we did not set the label and where we will see something like `jcrOAuthProvider.label.something` instead of a correct label?